src/utils.ts

```diff
@@ -274,6 +274,7 @@ const queryStringParser = (
 ): Dictionary<string> => {
     let urlParams: URLSearchParams | URLSearchParamsFallback;
     let results: Dictionary<string> = {};
+    let lowCaseUrlParams: Dictionary<string> = {};
```

Comment:
nit: Spell the full `lowerCaseUrlParams` out instead of abbreviating

Reply:
+1

Reply:
done